### DISCUSSION TITLE
What happened to ZStack?

### DISCUSSION THREAD

**Discussion by ts-candide** [gluestack/gluestack-ui#1708] [General]
### Description

`ZStack` is present in NativeBase, and was [present in this codebase at one stage](https://github.com/gluestack/gluestack-ui/commit/0f843a559a42a2aaa60ef91767f458c5c4760c62), but is no longer available. Can we have `ZStack` back?

### Problem Statement

I'd like to be able to align items to the z-axis, as we could using NativeBase.

### Proposed Solution or API

Can we reinstate the code from [this commit](https://github.com/gluestack/gluestack-ui/commit/0f843a559a42a2aaa60ef91767f458c5c4760c62)?

**Comment by Viraj-10** [comment]
Hey @ts-candide, `ZStack` just iterate through children and attaches `position:"absolute"`. While making `gluestack-ui` we remove such components which can be achieved by users easily but adds performance overhead. 
Thanks for reporting this we will add recipes in docs to this missing components so that migration is easier.

**Reply by ts-candide** [reply]
Thanks for your reply. In the meantime, could you give an example of how such a custom ZStack component could be built with GlueStack?

**Reply by Viraj-10** [reply]
you can use [this](https://github.com/gluestack/gluestack-ui/commit/0f843a559a42a2aaa60ef91767f458c5c4760c62) as reference.
